src/ralph/lib/transitions/models.py

```diff
@@ -238,10 +241,9 @@ def run_field_transition(
     """
     Execute all actions assigned to the selected transition.
     """
-    if not isinstance(instances, Iterable):
+    if not isinstance(instances, list):
```

Comment:
Tuple can not be modified.
As I wrote, we need an object that can be modified. The only such a list.

If my solution for the transition is not good, ask for suggestions for a better solution move bo to dc and dc to bo. There must be the possibility of redirects to the new objects from transitions.